crates/epaint/src/text/fonts.rs

```diff
@@ -727,13 +727,156 @@ struct GalleyCache {
 }
 
 impl GalleyCache {
+    fn layout_multiline(&mut self, fonts: &mut FontsImpl, job: LayoutJob) -> Galley {
```

Comment:
Nit: move this to under the main `layout` function (top->down ordering)

Reply:
Done.